iOS Client/PodcastsController/PodcastFilterTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PodcastFilterTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class PodcastFilterTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var toggleSwitch: UISwitch!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var podcastImage: UIImageView!
+    
+    var podcast: PodcastStream?
+    
+    static var reuseIdentifier: String = "PodcastFilterTableViewCell"
```

Comment:
interesting, coupling the reuse identifier with the cell class. I typically don't see this pattern. Since tableViews register a certain cell class (or nib) for a certain reuse identifier, it seems like you want control over what to make the reuse identifier. The cell usually doesn't prescribe it. Kind of makes sense in case multiple cells happened to have a namespace collision with their reuse identifiers. 

From a code reusability perspective, you may be happier down the road if this cell container is a little bit more generic and dumb without bundling in these details. 

As stylish as this is, I think its a bit dangerous. Of course, not going to fret about it too much for this project.

Reply:
Yeah honestly I don't have a strong opinion about reuse identifiers. I think on Synch we keep them with the cell class like I have here, but not too positive about the ramifications of that. I'll move them to properties on the VC to decouple